src/super_gradients/training/models/detection_models/pp_yolo_e/pp_yolo_e.py

```diff
@@ -165,6 +166,9 @@ def predict(
         conf: Optional[float] = None,
         batch_size: int = 32,
         fuse_model: bool = True,
+        target_bboxes: Optional[List[np.ndarray]] = None,
```

Comment:
same here about the type (can be a single ndarray as well)

Reply:
0798978413fab364fd7d89ea23baeab266c80e77